### ISSUE TITLE
2025-11-18: sgl-project/sglang 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#434]
# 每日更新报告（2025-11-18）

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-18 22:53:53 | sglang-bot | Add /tag-and-rerun-ci (#13521) |
| 2025-11-18 22:24:58 | Lianmin Zheng | Update docs (#13519) |
| 2025-11-18 21:42:31 | Lianmin Zheng | Trigger CI retry with edit (#13516) |
| 2025-11-18 21:29:15 | Junrong Lin | [Feature] Re:Enable hybrid mem saver (#12962) |
| 2025-11-18 21:23:05 | Lianmin Zheng | Add docs on trigger ci (#13513) |
| 2025-11-18 20:39:58 | Lianmin Zheng | Use slash command to trigger CI (#13512) |
| 2025-11-18 20:10:04 | Lianmin Zheng | README.md -> FOLDER_README.md (#13510) |
| 2025-11-18 20:00:15 | Lianmin Zheng | Add .github/CI_PERMISSIONS.json to define the CI permissions (#13509) |
| 2025-11-18 19:23:27 | alisonshao | fix: create git tags directly instead of temporary branches (#13168) |
| 2025-11-18 17:54:58 | Morpheus Guo | [BugFix] Accuracy and function Issue when run ptpc quant model (#13157) |
| 2025-11-18 17:34:23 | Zhiqiang Xie | [HiCache] Critical fix to host memory double free (#13501) |
| 2025-11-18 16:13:18 | Nicholas | [VLM][feat] Support encoder DP for Qwen2.5-VL (#13126) |
| 2025-11-18 15:53:33 | Liangsheng Yin | [CI] fix lint yml (syntax error) (#13496) |
| 2025-11-18 15:36:31 | Xiaoyu Zhang | [opt kimi k2 3/n] opt kimi_k2 moe_fused_gate kernel (#13374) |
| 2025-11-18 15:35:49 | Liangsheng Yin | [CI] Add input for pr-gate (#13491) |
| 2025-11-18 15:18:43 | Cheng Wan | fix: change performance log directory to cache path (#13482) |
| 2025-11-18 14:59:46 | Liangsheng Yin | [CI] remove auto-labeling `run-ci` label. (#13486) |
| 2025-11-18 14:24:58 | Liangsheng Yin | [CI] fix triggered by a non-run-ci label (#13393) |
| 2025-11-18 13:25:29 | alisonshao | CI: fix NFS EBUSY error in PR test workflow (#13460) |
| 2025-11-18 12:46:24 | b8zhong | [Piecewise CUDA Graph] Support ModelOpt FP8 (#13094) |
| 2025-11-18 12:21:57 | Liangsheng Yin | Fix lora test (#13479) |
| 2025-11-18 12:20:19 | Shangming Cai | [PD] Clarify init method docstrings for kvsender and kvreceiver (#13476) |
| 2025-11-18 12:18:50 | wingedge | chore: add an unified server arg for multimodal inputs preprocess config(#12149) |
| 2025-11-18 12:02:03 | Zijian Zhang | model: support JetVLM (#13289) |
| 2025-11-18 11:28:21 | Simo Lin | [model-gateway] use worker startup time out for worker registration (#13473) |
| 2025-11-18 10:58:17 | Binyao Jiang | [GLM4.6v] Relax the constraint of non-user role chat completion message schema for new GLM-v release (#13258) |
| 2025-11-18 10:58:00 | Binyao Jiang | [GLM4.6v] Required changes for bumping up to transformer 5.x (#13229) |
| 2025-11-18 10:42:35 | Douglas Yang | Adding CI Monitor Improvements (#13462) |
| 2025-11-18 10:32:59 | Yuhao Yang | [CI] re-enable test_vision_openai_server_a ci (#13444) |
| 2025-11-18 10:13:24 | Sai Enduri | [AMD CI] Update sgl-router python path in dockerfile. (#13458) |
| 2025-11-18 10:03:45 | Minglei Zhu | fix MambaPool clear method after refactoring (#13449) |
| 2025-11-18 09:44:31 | Baizhou Zhang | Add bfloat16 tuned fused moe config for Dpsk-MTP layer on B200 (#13455) |
| 2025-11-18 09:42:17 | pansicheng | Add interface_v1 option for dynamic HiCache backend (#13140) |
| 2025-11-18 09:08:02 | Kangyan-Zhou | Update pr-test.yml to fix invalid job name error |
| 2025-11-18 08:55:04 | Kangyan-Zhou | Temporarily comment out multimodal gen test to recover runners (#13463) |
| 2025-11-18 08:13:28 | Kaixi Hou | [NVIDIA] Fix broken fp8 MoE of deepseek v3 (#13264) |
| 2025-11-18 07:35:18 | Vedant V Jhaveri | [Embeddings Performance Testing] Add performance test for embedding models (#12359) |
| 2025-11-18 07:33:02 | Kangyan-Zhou | Fixes validation errors for Wan-AI models which store model weights in subdirectories (#13461) |
| 2025-11-18 05:26:56 | Kangyan-Zhou | Fix 8-gpu B200 nightly tests (#13457) |
| 2025-11-18 05:14:17 | Sirut Buasai | extend sagemaker.Dockerfile serve script to allow all sglang serve flags (#13173) |
| 2025-11-18 05:10:37 | Kangyan-Zhou | Add missing models (#13456) |
| 2025-11-18 04:34:36 | Simo Lin | Remove verbs from GET endpoint paths to follow REST standards (#13273) |
| 2025-11-18 04:20:49 | hlu1 | [Deepseek V3.2] Use torch.compile to speed up torch.cat in nsa (#13022) |
| 2025-11-18 02:35:13 | Liangsheng Yin | Temporarily disable model hooks CI (#13450) |
| 2025-11-18 02:08:19 | Liangsheng Yin | [2/N] CI refactor: sperate some backend-independent CPU tasks. (#13447) |
| 2025-11-18 00:56:30 | Mick | diffusion: fix loading with local model_path (#13445) |
| 2025-11-18 00:45:20 | Xiaoyu Zhang | [Ci tiny fix] Lower score threshold in evaluation test (#13443) |
